charts/vald/values.yaml

```
@@ -1058,7 +1058,7 @@ gateway:
       pathType: ImplementationSpecific
       # @schema {"name": "gateway.lb.ingress.enabled", "type": "boolean"}
       # gateway.lb.ingress.enabled -- gateway ingress enabled
-      enabled: false
+      enabled: true
```

```suggestion
      enabled: false
```
It would be better to set false. 🙏 If we do not have a controller for ingress, we cannot create an ingress resource and this deployment will fail.
Since I think values.yaml should work under any environment, it makes sense to let the user set it to enable if we want to use ingress.

---

I am assuming that if `ingress:false`, this setting is ignored in the first place. (If I'm wrong, please point it out.)
https://github.com/vdaas/vald/blob/main/charts/vald/templates/gateway/filter/ing.yaml#L17

I am thinking that if this flag is set to false, defaultBackend will behave differently in the new version, which may confuse existing users.

---

@hlts2 said this change forces the existing users who don't use `ingress` to change their values.
also, all tutorials don't apply the `ingress` feature.
That's way @hlts2 suggested do not change the `gateway.lb.ingress.enabled` value.

---

@vankichi Thanks for the additional information 🙏 
@takuyaymd I commented because there are cases where a user does not use Ingress, such as when running locally, so I thought it would be better to be able to enable the setting when needed 👍